readme.md

```diff
@@ -309,13 +308,14 @@ can map to different actions depending on context.
 | `GET`           | `/widgets`                        | List widgets        |
 | `POST`          | `/widgets/3rDrZRbdeSJJeR4jQSWAjK` | Invalid             |
 | `GET`           | `/widgets/3rDrZRbdeSJJeR4jQSWAjK` | Get a single widget |
+| `PATCH`         | `/widgets/3rDrZRbdeSJJeR4jQSWAjK` | Update a single widget |
 
 ### Headers
 
 The most basic header that API services must support, and that clients
 should provide, is the `Content-Type` header. In general, the client will
 supply this header with the value `application/json` to specify that the
-request body is JSON. However, API services must not require this header and
+request body is JSON. Multipart requests may use other appropriate values in this header. However, API services must not require this header and
```

Comment:
Can we change it to be "Multipart requests, for example, may use [...]"? I don't think this was disallowed as it was before, so since it's a clarification by example it's more clear to phrase it as such, IMO.